src/stylesheets/base/_base.scss

```diff
@@ -33,3 +32,10 @@ body {
   position: absolute;
   width: 1px;
 }
+
+ul li {
+  list-style: disc inside none;
+  position: relative;
+  left: -.1rem;
+  padding-left: 1rem;
+}
```

Comment:
you miiiight want to scope this style so it only applies to `<ul>`(s) in the speech bubbles

Reply:
yep moved to the _concept. scss.

resolved in pr#4